packages/stacked_generator/lib/src/generators/router/router_class_generator.dart

```diff
@@ -145,10 +149,11 @@ class RouterClassGenerator extends BaseGenerator {
 
     if (r.parameters?.isNotEmpty == true) {
```

Comment:
I never saw this shortcut, how does it work? I assume is the same as doing `r.parameters?.isNotEmpty ?? true`,  right?

Reply:
Me neither, this is 15 months old code, but I guess it should read like this 
1- if isNotEmpty is null then `null==true` which is false
2- if isNotEmpty is not null then it should execute normally

equals to
`r.parameters?.isNotEmpty ?? false`